File: Curso-de-POO/Python/account.py
class Account:
    id = int
    name = str
    document = str
    email = str
    password = str
    def __init__(self, name, document, email, password):
        self.name = name
        self.document = document
        self.email = email
        self.password = password


class User(Account):
    def __init__(self, name, document, email, password):
        super().__init__(name, document, email, password)


class Driver(Account):
    def __init__(self, name, document, email, password):
        super().__init__(name, document, email, password)

File: Curso-de-POO/PHP/account.php
<?php
class Account{
    public $id;
    public $name;
    public $document;
    public $email;
    public $password;

    public function __construct($name, $document, $email, $password)
    {
        $this->name = $name;
        $this->document = $document;
        $this->email = $email;
        $this->password = $password;
    }
}

class User extends Account{
    public function __construct($name, $document, $email, $password){
        parent::__construct($name, $document, $email, $password);
    }
}

class Driver extends Account{
    public function __construct($name, $document, $email, $password){
        parent::__construct($name, $document, $email, $password);
    }
}
?>